anime-list-master.xml

```diff
@@ -74,6 +74,9 @@
   </anime>
   <anime anidbid="15" tvdbid="79107" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Kanon</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-1;2-0;3-0;4-0;</mapping>
+    </mapping-list>
```

Comment:
Same season 1-1(;2-2;3-3;...) is implied, no need to specify it.
TVDB only has one special episode so the AniDB ones won't conflict with anything, there's no need to unmap them.
```suggestion

```